doc/api/events.markdown

```diff
@@ -377,6 +377,42 @@ listener array. If any single listener has been added multiple times to the
 listener array for the specified `event`, then `removeListener` must be called
 multiple times to remove each instance.
 
+Note that once an event has been emitted, all listeners attached to it at the
+time of emitting will be called in order. This implies that any `removeListener`
+call *after* emitting and *before* the last listener finishes execution will
+not affect the current listener array. Subsequent events will behave as expected.
+
+```js
+const myEmitter = new MyEmitter();
```

Comment:
There is a PR in place that validates code blocks in the docs per eslint rules.

Could you please add

```js
const EventEmitter = require('events');
class MyEmitter extends EventEmitter {}

const myEmitter = new MyEmitter();
```